release_notes/ocp_3_9_release_notes.adoc

```diff
@@ -0,0 +1,434 @@
+available. This release is based on
+link:https://github.com/openshift/origin/releases/tag/v3.7.0-rc.0[OpenShift
+Origin 3.9]. New features, changes, bug fixes, and known issues that pertain to
+{product-title} 3.9 are included in this topic.
+
+{product-title} 3.9 is supported on RHEL 7.3, 7.4.2, and Atomic Host 7.4.2 and
+newer with the latest packages from Extras, including Docker 1.12.
+
+For initial installations, see the
+xref:../install_config/install/planning.adoc#install-config-install-planning[Installing
+a Cluster] topics in the
+xref:../install_config/index.adoc#install-config-index[Installation and
+Configuration] documentation.
+
+To upgrade to this release from a previous version, see the
+xref:../install_config/upgrading/index.adoc#install-config-upgrading-index[Upgrading
+a Cluster] topics in the
+xref:../install_config/index.adoc#install-config-index[Installation and
+Configuration] documentation.
+
+[[ocp-39-new-features-and-enhancements]]
+== New Features and Enhancements
+
+This release adds improvements related to the following components and concepts.
+
+[[ocp-39-container-orchestration]]
+=== Container Orchestration
+
+[[ocp-39-crio]]
+==== CRI-O
+
+CRI-O v1.0 is a lightweight, native Kubernetes container runtime interface. By
+design, it provides only the runtime capabilities needed by the kubelet. CRI-O is
+designed to be part of Kubernetes and evolve in lock-step with the platform.
+
+CRI-O brings:
+
+* A minimal and secure architecture.
+* Excellent scale and performance.
+* The ability to run any Open Container Initiative (OCI) or docker image.
+* Familiar operational tooling and commands.
+
+image::crio-3-7.png[CRI-O]
+
+To install and run CRI-O alongside `docker`, set the following in the
+`[OSEv3:vars]` section
+xref:../install_config/install/advanced_install.adoc#configuring-ansible[Ansible inventory file] during cluster installation:
+
+----
+openshift_use_crio=true
+----
+
+This setting pulls the *openshift3/cri-o* system container image from the
+link:https://access.redhat.com/containers[Red Hat Registry] by default. If you
+want to use an alternative CRI-O system container image from another registry,
+you can also override the default using the following variable:
+
+----
+openshift_crio_systemcontainer_image_override=<registry>/<repo>/<image>:<tag>
+----
+
+[NOTE]
+====
+The `atomic-openshift-node` service must be RPM- or system container-based when
+using CRI-O; it cannot be `docker` container-based. The installer protects again
+using CRI-O with `docker` container nodes and will halt installation if
+detected.
+====
+
+When CRI-O use is enabled, it is installed alongside `docker`, which currently
+is required to perform build and push operations to the reigstry. Over time,
+temporary `docker` builds can accumulate on nodes. You can optionally set the
+following to enable garbage collection, which adds a daemonset to clean out the
+builds:
+
+----
+openshift_crio_enable_docker_gc=true
+----
```

Comment:
@ashcrow can you answer this question?

Reply:
It could be made default but the documentation in the inventory examples explain when and why one would want to enable it. TBH I don't have strong feelings either way.

Reply:
To clarify I don't have strong feelings about making it or keeping it from being default as I tend to think that needing to gc in this way probably won't be a long required thing.

Reply:
It should be a requirement for as long as we support docker (IMO), or as long as docker is installed on the nodes. 

Given that, should we just be enabling this (when we see docker installed on the node)?

Reply:
@sferich888 that's fair. Let's go ahead and make it default if the person is using crio as long as we assume they are building on crio nodes as well. Does that work?

Reply:
https://github.com/openshift/openshift-ansible/pull/7310

Reply:
The above PR merged. PTAL @sferich888.